lib/services/access_checker.go

```diff
@@ -389,7 +389,7 @@ func AccessInfoFromLocalIdentity(identity tlsca.Identity, access UserGetter) (*A
 			return nil, trace.Wrap(err)
 		}
 
-		log.Warnf("Failed to find roles or traits in x509 identity for %v. Fetching	"+
+		log.Warnf("Failed to find roles in x509 identity for %v. Fetching "+
```

Comment:
I've updated this warning as it is misleading. The check here used to be for no roles or traits, but now only checks for a lack of roles.